deploy.config.js

```diff
@@ -24,6 +25,7 @@ exports.providers = {
         },
         mainnet: {
             lendingPoolAddressProvider: "0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5",
+            dataProvider: "0x057835Ad21a177dbdd3090bB1CAE03EaCF78Fc6d",
```

Comment:
we should remove `dataProviderId` on line 44 below

Reply:
alright

Reply:
pushed